### PR TITLE
fix server-gw links

### LINKED ISSUE #45: Add download link for MediaPlayer Kit on the SDK page
Name: Agora MediaPlayer Kit

Download link:

- Android v1.3.0: https://download.agora.io/sdk/release/Agora_Media_Player_for_Android_rel.v1.3.0_14564_ffmpeg_player_lite_20210723_1234.zip

- iOS v1.3.0: https://download.agora.io/sdk/release/Agora_Media_Player_for_iOS_rel.v1.3.0_53009_ffmpeg_player_lite_20210702_1606.zip

- macOS v1.3.0: https://download.agora.io/sdk/release/Agora_Media_Player_for_Mac_rel.v1.3.0_31754_ffmpeg_player_lite_20210716_1813.zip

- Windows (x86) v1.3.0: https://download.agora.io/sdk/release/Agora_Media_Player_for_Windows_x86_rel.v1.3.0_63393_ffmpeg_player_lite_20210727_1117.zip

- Windows (x64) v1.3.0: https://download.agora.io/sdk/release/Agora_Media_Player_for_Windows_x64_rel.v1.3.0_63392_ffmpeg_player_lite_20210727_1117.zip

### PR DESCRIPTION
**Pull request by Kishan-Dhakan**
- Resolves https://github.com/AgoraIO/Docs-Source/issues/555
- Resolves https://github.com/AgoraIO/Docs-Source/issues/643
- Resolves #45

### CODE REVIEW

File: data/sdks.js [hunk 1]
```diff
@@ -813,18 +813,18 @@ export const data = {
...
+              label: 'Version 3.8.2 for C++',
               downloadLink:
-                'https://docs.agora.io/en/server_gateway/server_gateway_run_demo?platform=Linux',
+                'https://download.agora.io/sdk/release/Agora-RTC-x86_64-linux-gnu-v3.8.202.20-20220627_152601-214165.tgz',
               packageManager: '#',
             },
             {
               id: 'java-cloud-gateway-sdk-linux',
               icon: '',
               label: 'Version 3.0.6 for Java',
```
Comment (billy-the-fish) [inline data/sdks.js:824]: Check the version number on maven please.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Can you align the version number please.

File: data/sdks.js (hunk 1 above)
Comment (Kishan-Dhakan) [reply data/sdks.js]: Resolved

**Review by billy-the-fish — APPROVED** [review]
LGTM

**Pull request merged**